frontend/src/scene/main-view/CourseModal.js

```diff
@@ -133,14 +173,68 @@ class CourseModal extends React.Component {
                                     this.props.courseStore.reserveCourse(course)
                                 }
                             >
-                                Varaa ja Maksa
+                                {i18nContent.courseDescriptionForm.dismiss}
                             </Button>
                         </PaymentSection>
                     </Fragment>
                 )}
+                {reservedCourse && (
```

Comment:
It's better to make a separate modal for this

Reply:
created separate modal